api/test/test_helper/fake_android_plugin.gd

```diff
@@ -27,9 +27,16 @@ extends Object
 var _asset_location_store : Dictionary
 var _asset_pack_location_store : Dictionary
 
+var _asset_pack_states_store : Dictionary
```

Comment:
I used the naming of ```_asset_pack_states_store``` since this variable is going to store all AssetPackStates relevant to ```fetch()``` or ```cancel()``` regardless of their current status.